types/stampit/stampit-tests.ts

```diff
@@ -24,22 +51,19 @@ foo.getB(); // "b"
 // Here's a mixin with public methods, and some refs:
 const membership = stampit({
     methods: {
-        members: {},
+        // members: {},
         add(member: any) {
             this.members[member.name] = member;
             return this;
         },
         getMember(name: any) {
             return this.members[name];
         }
-    },
-    refs: {
-        members: {}
     }
 });
 
 // Let's set some defaults:
-const defaults = stampit().refs({
+const defaults = stampit().conf({
```

Comment:
The v3.0 `refs` was renamed to `props` in v4.0. Sorry for the confusion. In these tests please rename `refs` to `props`. And `props` to `deepProps`. Here is the breaking changes release notes: https://github.com/stampit-org/stampit/releases/tag/v4.0.0